content/es/docs/tutorials/_index.md

```diff
@@ -31,6 +31,8 @@ Antes de recorrer cada tutorial, recomendamos añadir un marcador a
 
 ## Configuración
 
+* [Example: Configuring a Java Microservice](/docs/tutorials/configuration/configure-java-microservice/)
```

Comment:
Me parece que debería estar traducido y  aprovechar en traducir la siguiente linea
```suggestion
* [Ejemplo: Configurando un Microservicio en Java](/docs/tutorials/configuration/configure-java-microservice/)
```

Reply:
@electrocucaracha  done, thank you